src/vue/common/FormConfirmation.spec.js

```diff
@@ -0,0 +1,108 @@
+import FormConfirmation from './FormConfirmation'
+import { mount } from '@vue/test-utils'
+import i18next from 'i18next'
+import { i18nOptions } from '@/i18n'
+
+describe('FormConfirmation component test', () => {
+  beforeEach(() => {
+    i18next.init(i18nOptions)
+  })
+
+  it('template properly renders the data provided through props', () => {
+    const message = 'Please ensure everything is OK!'
+    const okButtonText = 'I\'m okay, let\'s go!'
+    const cancelButtonText = 'WOWO, wait please!'
+
+    const wrapper = mount(FormConfirmation, {
+      propsData: {
+        message,
+        okButtonText,
+        cancelButtonText
+      }
+    })
+
+    const renderedMessage = wrapper
+      .find('.form-confirmation__msg')
+      .text()
+    const renderedOkButtonText = wrapper
+      .find('.form-confirmation__ok-btn')
+      .text()
+    const renderedCancelButtonText = wrapper
+      .find('.form-confirmation__cancel-btn')
+      .text()
+
+    expect(renderedMessage).to.equal(message)
+    expect(renderedOkButtonText).to.equal(okButtonText)
+    expect(renderedCancelButtonText).to.equal(cancelButtonText)
+  })
+
+  it('properly renders the default data if no props provided', () => {
+    const message = 'Please recheck the form before confirmation'
```

Comment:
it’s not so clear what’s the source of the message you expect here. Please mock the translation data right in the way you did in i18next testing to explicitly declare the source of data